src/rpc/mining.cpp

```diff
@@ -280,7 +281,7 @@ UniValue prioritisetransaction(const JSONRPCRequest& request)
     CAmount nAmount = request.params[2].get_int64();
 
     if (!(request.params[1].isNull() || request.params[1].get_real() == 0)) {
-        throw JSONRPCError(RPC_INVALID_PARAMETER, "Priority is not supported, and adjustment thereof must be zero.");
+        throw JSONRPCError(RPC_INVALID_PARAMETER, "Priority is no longer supported, first argument to prioritisetransaction must be 0.");
```

Comment:
Using named arguments, "first argument" may be confusing. Also suggest the recommendation should be null, not 0.